### PR TITLE
Use the fetch types from serverlet

### PR DESCRIPTION
**Pull request by swansontec**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,15 +32,16 @@
...
-    "lint": "eslint --ext .js,.ts .",
-    "precommit": "lint-staged && npm-run-all -p test build.*",
-    "prepare": "npm-run-all clean -p build.*",
-    "test": "nyc mocha test/**/*.ts"
+    "fix": "eslint . --fix",
+    "flow": "flow",
+    "lib": "rollup -c",
+    "lint": "eslint .",
+    "precommit": "lint-staged && npm-run-all test lib flow types",
+    "prepare": "npm-run-all clean -p lib types",
```
Comment (samholmes) [inline package.json:41]: The prepare script is missing `flow`: 

```
    "prepare": "npm-run-all clean -p lib types flow",
```

**Review by samholmes — APPROVED** [review]
I align with these changes, though we must make sure these type changes don't break the browser/react-native environments. Testing this branch in the core within the GUI would be a necessary measure to take if not already.

**Comment by swansontec** [comment]
Created PR #12 to address the logging security issues.

**Comment by samholmes** [comment]
Testing on the sync server is a moot point given the sync-client dependency is already behind by a major version bump.

**Pull request merged**